Tasks/NuGetCommandV2/_buildConfigs/Node20/package.json

```diff
@@ -18,8 +18,8 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
     "@types/node": "^20.3.1",
-    "azure-pipelines-task-lib": "^4.0.0-preview",
```

Comment:
Did you intentionally lock this to exactly 4.0.0-preview?

Reply:
yep, there are some breaking (test cases) changes from preview to later version.  It is fixable but don't have the time capacity for it now.